numba/core/codegen.py

```diff
@@ -60,8 +62,369 @@ class _CFG(object):
+        cs = defaultdict(lambda: 'white') # default bg colour is white
+        cs['marker'] = 'orange'
+        cs['python'] = 'yellow'
+        cs['truebr'] = 'green'
+        cs['falsebr'] = 'red'
+        cs['incref'] = 'cyan'
+        cs['decref'] = 'turquoise'
+        cs['raise'] = 'lightpink'
+        cs['meminfo'] = 'lightseagreen'
+        cs['return'] = 'purple'
+
+        # Get the raw dot format information from LLVM and the LLVM IR
+        fn = self.cres.get_function(self.name)
+        #raw_dot = ll.get_function_cfg(fn).replace('\\l...', '')
+        llvm_str = self.cres.get_llvm_str()
+
+        def get_metadata(llvm_str):
+            """ Gets the metadata entries from the LLVM IR, these look something
+            like '!123 = INFORMATION'. Returns a map of metadata key to metadata
+            value, i.e. from the example {'!123': INFORMATION}"""
+            md = {}
+            metadata_entry = re.compile(r'(^[!][0-9]+)(\s+=\s+.*)')
+            for x in llvm_str.splitlines():
+                match = metadata_entry.match(x)
+                if match is not None:
+                    g = match.groups()
+                    if g is not None:
+                        assert len(g) == 2
+                        md[g[0]] = g[1]
+            return md
+
+        md = get_metadata(llvm_str)
+
+        # setup digraph with initial properties
+        def init_digraph(name, fname, fontsize):
+            # name and fname are arbitrary graph and file names, they appear in
+            # some rendering formats, the fontsize determines the output
+            # fontsize.
+            f = gv.Digraph(name, fname)
+            f.attr(rankdir='TB')
+            f.attr('node', shape='none', fontsize='%s' % str(fontsize))
+            return f
+
+        f = init_digraph(self.name, self.name, fontsize)
+
+        # A lot of regex is needed to parse the raw dot output. This output
+        # contains a mix of LLVM IR in the labels, and also DOT markup.
+
+        # DOT syntax, matches a "port" (where the tail of an edge starts)
+        port_match = re.compile('.*{(.*)}.*')
+        # DOT syntax, matches the "port" value from a found "port_match"
+        port_jmp_match = re.compile('.*<(.*)>(.*)')
+        # DOT syntax, matches the DOT object (node), captures the entire node
+        node = re.compile(r'.*Node0x.*=".*', re.DOTALL|re.MULTILINE)
+        # LLVM syntax, matches a LLVM debug marker
+        metadata_marker = re.compile(r'.*!dbg\s+(![0-9]+).*')
+        # LLVM syntax, matches a location entry
+        location_expr = (r'.*!DILocation\(line:\s+([0-9]+),'
+                         '\s+column:\s+([0-9]),.*')
+        location_entry = re.compile(location_expr)
+        # LLVM syntax, matches LLVMs internal debug value calls
+        dbg_value = re.compile(r'.*call void @llvm.dbg.value.*')
+        # LLVM syntax, matches tokens for highlighting
+        nrt_incref = re.compile("NRT_incref")
```

Comment:
make the pattern a bit more restrictive
```suggestion
        nrt_incref = re.compile(r"@NRT_incref\b")
```